pkg/sql/revert_test.go

```diff
@@ -40,7 +42,15 @@ func TestRevertTable(t *testing.T) {
 	ctx := context.Background()
 
 	s, sqlDB, kv := serverutils.StartServer(
-		t, base.TestServerArgs{UseDatabase: "test"})
+		t,
+		base.TestServerArgs{
+			Knobs: base.TestingKnobs{
+				Server: &server.TestingKnobs{
+					DisableAutomaticVersionUpgrade: 1,
+					BinaryVersionOverride:          clusterversion.ByKey(clusterversion.PreventNewInterleavedTables - 1),
```

Comment:
Is this change intentional? Shouldn't you instead be removing the `"interleaved"` subtest?